package.json

```diff
@@ -52,5 +52,8 @@
     "jest": "^26.6.3",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.2.1"
+  },
+  "devDependencies": {
```

Comment:
Just a little nit, but what do you think about adding here `test:e2e` and `test:accesibility` so we have those commands exposed in the root of the repo?

Reply:
yeah i can add those to the top

Reply:
I pushed them all to the root and cleaned up unused commands.

Reply:
Perfect thanks!!